google/cloud/forseti/services/action_engine.proto

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+service ActionEngine {
+  rpc Ping(PingRequest) returns (PingReply) {
+  }
+
+  rpc List(ListRequest) returns (ListResponse) {
+  }
+
+  rpc ProcessResults(ProcessResultsRequest)
+      returns (stream ProcessResultsResponse) {
+  }
+}
+
+message PingRequest {
+  string data = 1;
+}
+
+message PingReply {
+  string data = 1;
+}
+
+message ListRequest {
+}
+
+message Action {
+  string action_id = 1;
+  enum ActionType {
+    BaseAction = 0;
+    SampleAction = 1;
+  }
+  ActionType type = 2;
+  repeated string triggers = 3;
+  // The string representation of the configuration.
+  string config = 4;
+}
+
+message ListResponse {
+  repeated Action actions = 1;
+}
+
+message RuleResult {
+  // The id of the rule, must be unique
+  string rule_id = 1;
+  // The result of the rule.
+  bool result = 2;
```

Comment:
What does this bool represent? What does true mean, or false?

Reply:
Removed based on discussion.